docs/user-guide.md

```diff
@@ -211,6 +211,10 @@ The currently supported options are:
 | kompose.cronjob.schedule                            | kubernetes cronjob schedule (for example: '1 * * * *')                               |
 | kompose.cronjob.concurrency_policy                  | 'Forbid' / 'Allow' / 'Never' / ''                                                    |
 | kompose.cronjob.backoff_limit                       | kubernetes cronjob backoff limit (for example: '6')                                  |
+| kompose.hpa.minreplicas                             | defines Horizontal Pod Autoscaler min pod replicas                                   |
+| kompose.hpa.maxreplicas                             | defines Horizontal Pod Autoscaler max pod replicas                                   |
```

Comment:
we need to follow how we do the other labels.

this should be instead: `kompose.hpa.replicas.min` and `kompose.hpa.replicas.max`